ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -1044,6 +1044,15 @@ class SignalProducerSpec: QuickSpec {
 
 					expect(completed).to(beTruthy())
 				}
+
+				it("should not deadlock") {
```

Comment:
Is this diagnose-able if it fails in the future? E.g. will Xcode timeout and kill the test with some meaningful error?